targets/realtek-rtl838x

```diff
@@ -0,0 +1,5 @@
+-- D-Link
+
+device('d-link_dgs-1210-10p', 'd-link_dgs-1210-10p', {
```

Comment:
Is this device name correct? For most targets it should be impossible for an underscore to appear.